gtest/src/state/accounts.rs

```diff
@@ -178,15 +189,15 @@ impl Accounts {
 
     // Clears accounts storage.
     pub(crate) fn clear() {
-        ACCOUNT_STORAGE.with_borrow_mut(|storage| {
-            storage.clear();
-            init_default_accounts(storage);
+        storage().with(|storage| {
+            storage.data_mut().clear();
+            init_default_accounts(&mut storage.data_mut());
```

Comment:
```suggestion
            init_default_accounts(storage.data_mut());
```

Reply:
`storage.data_mut()` returns `RefMut<'_, T>`, so to access the `&mut T`, we should deref mut the `RefMut`. And I do that by taking unique ref on it